docs/source/intro/install_system.rst

```diff
@@ -0,0 +1,278 @@
+
+           $ sudo apt install \
+              git python3-dev python3-pip \
+              postgresql postgresql-server-dev-all postgresql-client rabbitmq-server
+
+   .. tabbed:: MacOS X (Homebrew)
+
+       The recommended installation method for Mac OS X is to use `Homebrew <https://brew.sh>`__.
+
+       #. Follow `this guide <https://docs.brew.sh/Installation>`__ to install Homebrew on your system if not installed yet.
+
+       #. Open a terminal and execute:
+
+          .. code-block:: console
+
+              $ brew install postgresql rabbitmq git python
+              $ brew services start postgresql
+              $ brew services start rabbitmq
+
+   .. tabbed:: Windows Subsystem for Linux
+
+      *The following instructions are for setting up AiiDA on WSL 1/2 in combination with Ubuntu.*
+
+      #. Installing RabbitMQ:
+
+         * (WSL 1) Install and start the `Windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_.
+
+         * (WSL 2) Install RabbitMQ inside the the WSL:
+
+            .. code-block:: console
+
+                $ sudo apt install rabbitmq-server
+                $ sudo service rabbitmq-server start
+
+      #. Install Python and PostgreSQL:
+
+         .. code-block:: console
+
+             $ sudo apt install \
+                postgresql postgresql-server-dev-all postgresql-client \
+                git python3-dev python-pip
+             $ sudo service postgresql start
+
+      .. dropdown:: How to setup WSL to automatically start services after system boot.
+
+          Create a file ``start_aiida_services.sh`` containing the following lines:
+
+          .. code-block:: console
+
+             service postgresql start
+             service rabbitmq-server start # Only for WSL 2!
+
+          and store it in your preferred location, e.g., the home directory.
+          Then make the file executeable, and editable only by root users with:
+
+          .. code-block:: console
+
+             $ chmod a+x,go-w /path/to/start_aiida_services.sh
+             $ sudo chown root:root /path/to/start_aiida_services.sh
+
+          Next, run
+
+          .. code-block:: console
+
+             $ sudo visudo
+
+          and add the line
+
+          .. code-block:: sh
+
+             <username> ALL=(root) NOPASSWD: /path/to/start_aiida_services.sh
+
+          replacing ``<username>`` with your Ubuntu username.
+          This will allow you to run *only* this specific ``.sh`` file with ``root`` access (without password), without lowering security on the rest of your system.
+
+          Now you can use the Windows Task Scheduler to automatically execute this file on startup:
+
+          #. Open Task Scheduler.
+
+          #. In the "Actions" menu, click "Create Task".
+
+          #. In "General/Security options", select "Run whether user is logged on or not".
+
+          #. In the "Triggers" tab, click "New...".
+
+             #. In the "Begin the task:" dropdown, select "At startup".
```

Comment:
I've tested this for a bit now, and think it's not 100% bulletproof.. maybe because WSL may not be started when the task is trying to execute. Previously I had used `"At log on"`, which seemed to work fine. One could also set both triggers.

Not sure which is the best one to recommend here 🤔

Reply:
I say, let's keep it *as is* for now (it's at least not worse than the previous instructions) and we can update them once we know more.

Reply:
Yeah, agreed. If it ends up being a common source of trouble we can always revisit it.